packages/viewer/src/components/viewer-markup-arrow/interactions.ts

```diff
@@ -18,7 +18,8 @@ export class ArrowMarkupInteractionHandler extends MarkupInteractionHandler {
   public constructor(
     private readonly markupEl: HTMLVertexViewerMarkupArrowElement,
     private readonly editBegin: EventEmitter<void>,
-    private readonly editEnd: EventEmitter<void>
+    private readonly editEnd: EventEmitter<void>,
+    private readonly markupUpdated: EventEmitter<HTMLVertexViewerMarkupArrowElement>
```

Comment:
Thoughts on leveraging the `editEnd` for this event in the arrow, circle, and freeform elements? Looks like that's the only place we're dispatching the new event, so we might just be able to change the `EventEmitter<void>` to be a `EventEmitter<HTMLVertexViewerMarkupArrowElement>` (or other element types) and avoid the new event on the individual pieces of markup

Reply:
This was poorly named, since editEnd was dispatched when both creating and editing markup. I changed the name of the event to _interactionEnd_ and it now returns both the current piece of markup as well as a boolean describing if the markup was just created.

I think this makes it much easier to read now!